ZUC.cpp

```diff
@@ -156,12 +156,7 @@ unsigned int F(void) {
     return W;
 }
 
-#define MAKEU31(a, b, c) ( \
-    ((unsigned int)((unsigned int)(0) \
-    | (unsigned char)(a)) << 23) \
-    | ((unsigned int)(b) << 8) \
-    | (unsigned int)((unsigned int)(0) \
-    | (unsigned char)(c)))
+#define MAKEU31(a, b, c) (((unsigned int)((unsigned int)(0) | (unsigned char)(a)) << 23) | ((unsigned int)(b) << 8) | (unsigned int)((unsigned int)(0) | (unsigned char)(c)))
```

Comment:
Please leave it as is, your style is not readable

Reply:
your style was just not recognized by HLS tools .
but it's more readable as a c code. (y)